gallery/src/styles/NxFormSelect/NxFormSelectExample.tsx

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+const NxFormSelectExample = () =>
```

Comment:
Couldn't these examples be plain HTML?

Reply:
> Couldn't these examples be plain HTML?

They could, I've just gotten used to making the examples in React, it's good practice for me. I can switch them to HTML if you think that it matters.